app/containers/paperShow/index.tsx

```diff
@@ -482,16 +485,27 @@ class PaperShow extends React.PureComponent<PaperShowProps, PaperShowStates> {
       };
     });
 
+    const publisherForStructuredData = {
+      "@type": "Organization",
+      name: "Scinapse",
+      logo: {
+        "@type": "ImageObject",
+        url: "https://s3.amazonaws.com/pluto-asset/scinapse/scinapse-logo.png",
```

Comment:
There is another URL that uses CDN(CloudFront).
`https://assets.pluto.network`.